public/js/app/play.js

```diff
@@ -0,0 +1,54 @@
+import {Octokit} from "https://esm.sh/@octokit/core";
```

Comment:
Maybe you missed that this file is executed in a browser. Just used the recommended way for browsers.

Reply:
I guess the only potential discomfort I have here is about the presence of the script at the url and potential for injection.

Although to get around this we would have to use some bundler like vite or webpack correct

Reply:
even if we would have a builder, it will not change the potential harm.
if someone gets access to the repository (to any package we use), we can't do anything about it (forking and self-hosting would be an option), but since it's an official [sdk](https://github.com/octokit/octokit.js) from github, i don't have any discomfort.
there are other "security" issues looking into the code.

i would say, if we don't trust github or [esm](https://github.com/esm-dev/esm.sh), we should also not trust cloudflare or google-tagmanager/analytics^^

The github client used in node  was even more critical, since it could execute code in the container, this combined with node running as root and no checksum checks (already fixed with `--frozen-lockfile` option and the new Dockerfile) could have been more dramatic than just me ranting and creating a PR to fix that.

We also have to balance security, if the site would get compromised, they could only get a potential login-password, or impersonate FAF to trick the user to download/install/accept something that is not from us, if the browser-security is set to be wide open.

i would "trust" esm and github (maintainers of the repo), but maybe i am missing something here.

Reply:
I did not realize it was a github official sdk

Reply:
Was also surprised by the name